integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java

```diff
@@ -176,21 +187,6 @@ public void testOperatorUpgradeFrom3_0_3(@Namespaces(3) List<String> namespaces)
     upgradeOperator("3.0.3", OLD_DEFAULT_EXTERNAL_SERVICE_NAME_SUFFIX, true);
   }
 
-  /**
```

Comment:
Why are we removing this method ?

Reply:
what are you talking about, it's still there

  @Test
  @DisplayName("Upgrade Operator from 3.0.3 to latest")
  public void testOperatorUpgradeFrom3_0_3(@Namespaces(3) List<String> namespaces) {
    this.namespaces = namespaces;
    upgradeOperator("3.0.3", OLD_DEFAULT_EXTERNAL_SERVICE_NAME_SUFFIX, true);
  }

Reply:
you are removing testOperatorUpgradeFrom3_0_4, add it back

Reply:
done